pages/installation/os/centos.rst

```diff
@@ -123,6 +123,27 @@ Further reading
 * https://www.digitalocean.com/community/tutorials/an-introduction-to-selinux-on-centos-7-part-1-basic-concepts
 
 
+Multiple Server Setup
+---------------------
+
+If you plan to have multiple server taking care of different roles in your cluster :ref:`like we have in this big production setup <big_production_setup>` you need to modify only a few settings. Some of them already covered in our :ref:`Multi-node Setup guide<configure_multinode>` but we will refer them here to help you with this. The :ref:`default file location guide <default_file_location>` will give you the file you need to modify in your setup. 
+
+Elasticsearch
+^^^^^^^^^^^^^
+
+The Elasticsearch servers need on IP that can be reached over network set in ``network.host`` and all participants of the cluster in ``discovery.zen.ping.unicast.hosts``. That is enough to have a minial cluster setup.
```

Comment:
`discovery.zen.ping.unicast.hosts` only has to list some seed nodes, not all Elasticsearch nodes.